metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorDatabaseSpec.groovy

```diff
@@ -41,160 +40,174 @@ import spock.lang.Unroll
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
         when:
-        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive","testdb"))
+        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"))
         then:
-        client.getDatabase(_) >> {throw new NoSuchObjectException()}
+        client.getDatabase(_) >> { throw new NoSuchObjectException() }
         result == false
     }
 
-    def "Test for exist database TException"(){
+    def "Test for exist database TException"() {
         def client = Mock(MetacatHiveClient)
-        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter( new HiveTypeConverter() ) )
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
         when:
-        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive","testdb"))
+        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"))
         then:
-        client.getDatabase(_) >> {throw new TException() }
+        client.getDatabase(_) >> { throw new TException() }
         thrown ConnectorException
     }
 
     @Unroll
-    def "Test for listNames database with Exceptions"(){
+    def "Test for listNames database with Exceptions"() {
         def client = Mock(MetacatHiveClient)
-        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter( new HiveTypeConverter() ) )
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
 
         when:
-        def dbs = hiveConnectorDatabaseService.listNames(connectorContext, QualifiedName.ofDatabase("testhive","testdb"), null, new Sort(null, SortOrder.ASC), null )
+        def dbs = hiveConnectorDatabaseService.listNames(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"), null, new Sort(null, SortOrder.ASC), null)
         then:
-        1 * client.getAllDatabases() >> {throw exception}
+        1 * client.getAllDatabases() >> { throw exception }
         thrown result
 
         where:
-        exception                    | result
-        new TException()             |ConnectorException
-        new MetaException()          |InvalidMetaException
+        exception           | result
+        new TException()    | ConnectorException
+        new MetaException() | InvalidMetaException
 
     }
 
-    @Ignore
-    def "Test for get database" (){
+    def "Test for get database not found"() {
```

Comment:
I can't think of any case where the getdatabase returns null. The case with a null as the returned result isn't right. We should remove this case. @ajoymajumdar If I miss anything here, please point out.